recipes/scikit-ued/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "1.0.0.0" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "39293b55d609512949137f99d6534ae5981b0cab0169e30f029e8c21d91d912e" %}
+
+package:
+  name: 'scikit-ued'
+  version: '{{ version }}'
+
+source:
+  fn: 'scikit-ued-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/s/scikit-ued/scikit-ued-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python
+  number: 1
```

Comment:
In `staged-recipes`, the build `number` should always be `0`.

Reply:
Sounds good. I was trying to trigger a new build because of some connection timeout.